src/kernel/arch/aarch64/rpi.zig

```diff
@@ -47,6 +48,7 @@ pub inline fn turnOnLed() void {
 }
 
 pub inline fn turnOffLed() void {
+    // do not use zig here in order to side-step occasional compiler defects
```

Comment:
What compiler defects have you encountered in `turnOffLed` and `turnOnLed`?

Reply:
There were some alignment faults some time ago, but they have not recurred since enabling the mmu. I have removed this unneeded code.